validator/pandora/service_test.go

```diff
@@ -0,0 +1,68 @@
+package pandora
+
+import (
+	"context"
+	"github.com/prysmaticlabs/prysm/shared/testutil/require"
+	logTest "github.com/sirupsen/logrus/hooks/test"
+	"strings"
+	"testing"
+	"time"
+)
+
+// MockPandoraService method returns a mocked pandora service using
+// mocked rpc client and mocked rpc server.
```

Comment:
I think that only rpc server should be mocked

Reply:
Here i used a server with 3 mocked apis and the server is connected using `InProc`  communication.